products/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+class Category(models.Model):
+	name = models.CharField(max_length=254)
+	friendly_name = models.CharField(max_length=254, null=True, blank=True)
+
+	def __str__(self):
+		return f'{self.name}'
+
+	def get_friendly_name(self):
+		return self.friendly_name
+
+class Product(models.Model):
+	category = models.ForeignKey('Category', null=True, blank=True, on_delete=models.SET_NULL)
+	sku = models.CharField(max_length=254, null=True, blank=True)
```

Comment:
I agree. This stuff was added as an afterthought and Brian and I felt adding size and sku as a separate model (or models) would add a lot of complexity especially since products don't have "sizes" in the fixtures. In the videos I've recorded on this, I specifically noted that this is not ideal and that in a real ecommerce situation we would want to separate this stuff out and have an inventory tracking system and such, but that this will mimic the functionality on a basic level.

I do agree with you though, 100%. As part of a larger refactor we could absolutely add this.